book/tutorials/cloud-computing/atl08_parquet_files/atl08_parquet.ipynb

```diff
@@ -0,0 +1,300 @@
+    "    bounding_box=(-90,-56,-32,14),\n",
+    "    count=-1\n",
+    ")\n",
+    "year_month = f\"year={start.year}/month={start.month}\"\n",
+    "week = 0\n",
+    "len(results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7f82d0e1-c0d0-4165-ba97-63c8a890ef94",
+   "metadata": {},
+   "source": [
+    "## 4. Sort the results and setup the parquet"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d1e3632-7af4-4c3e-aa68-4dd11fc8d06e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sorted_results = sorted(results, key=lambda r : datetime.strptime(r['umm']['TemporalExtent']['RangeDateTime']['BeginningDateTime'], '%Y-%m-%dT%H:%M:%S.%fZ'))\n",
+    "\n",
+    "template_file = s3.open(sorted_results[0].data_links(access=\"direct\")[0], 'rb')\n",
+    "\n",
+    "atl08_parquet = aph.ParquetTable(\n",
+    "    geometadata_file='atl08-parquet-metadata.json',\n",
+    "    template_file=template_file\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "18f17bed-fb90-4386-9b47-05ea56b920a6",
+   "metadata": {},
+   "source": [
+    "## 5. Write results to the parquet table\n",
+    "\n",
+    "Write results to 1 parquet file, using the year-month as a partition. Later on if we add more weeks we can add them to new parquet files and new partitions as appropriate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42bc69f4-69d5-4682-be3d-02fdfeebea02",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%time\n",
+    "# fetch a group and write them to a partition\n",
+    "directory=\"atl08_parquet\"\n",
+    "os.makedirs(f\"{directory}/{year_month}\", exist_ok=True)\n",
+    "# i think it can only go one beam at a time even with more workers because of the global hdf5 interpreter lock\n",
+    "atl08_parquet.write_results_by_partition(sorted_results, s3, parquet_file=f\"{directory}/{year_month}/{week}.parquet\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6c41ee4f-5885-475b-930d-ad523f03bb7c",
+   "metadata": {},
+   "source": [
+    "## We're done creating the parquet!\n",
+    "\n",
+    "Now we can checkout the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "852324eb-e248-475f-a764-149c324891f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = pq.ParquetDataset(\"atl08_parquet\", partitioning=\"hive\", filters=[('year', '>=', 2021),\n",
+    "                                                     ('year', '<=', 2021),\n",
+    "                                                     ('month', '>=', 11),\n",
+    "                                                     ('month', '<=', 11)])\n",
+    "table = dataset.read(columns=[\"h_canopy\", \"geometry\"])\n",
+    "df = table.to_pandas()\n",
+    "df['geometry'] = df['geometry'].apply(wkb.loads)\n",
+    "\n",
+    "\n",
+    "gdf = gpd.GeoDataFrame(df, geometry='geometry')\n",
```

Comment:
```suggestion
    "gdf = gpd.GeoDataFrame(df, geometry='geometry', crs='EPSG:4326')\n",
```

(to avoid UserWarning: No CRS exists on data. from lonboard)